2996_reflection/reflection.md

```diff
@@ -972,7 +974,9 @@ struct universal_formatter {
 
   template <typename T>
   auto format(T const& t, auto& ctx) const {
-    auto out = std::format_to(ctx.out(), "{}@{@{", name_of(^T));
+    std::string_view name = has_identifier(^T) ? identifier_of(^T)
+                                               : "(unnamed-type)";
+    auto out = std::format_to(ctx.out(), "{}@{@{", type_label);
```

Comment:
Should the variable have been named `type_label`? Would rather just put the whole expression into the call to `format_to`.

Reply:
Done.